### ISSUE TITLE
export {a as default} uses module.exports but only exports is imported

### ISSUE THREAD

**Issue by lukeapage** [babel/babel#1260]
```
var a = 1;
export {a as default};
```

with export to amd non-strict is becoming

```
define(["exports"], function (exports) {
  "use strict";

  var a = 1;
  module.exports = a;
});
```

When I first tested your bugfix I'm sure it was working fine, so I think its a regression. I'll try some versions of babel.

**Comment by lukeapage** [comment]
this is also broken in 5.0 so either I didn't test your experiemental branch properly or it happened after your initial fix and the release of 5.0

**Comment by lukeapage** [comment]
wow, thats quick ;)

**Comment by sebmck** [comment]
Fixed as of 5.1.9, thanks!

**Comment by lukeapage** [comment]
many thanks.
